inspirehep/modules/workflows/views/callback.py

```diff
@@ -170,55 +263,32 @@ def robotupload_callback():
     If robotupload encountered an error sends an email
     to site administrator informing him about the error.
     """
+
     request_data = request.get_json()
     workflow_id = request_data.get('nonce', '')
     results = request_data.get('results', [])
-    response = {}
+    responses = {}
     for result in results:
         recid = int(result.get('recid'))
 
-        if recid in response:
+        if recid in responses:
+            # this should never happen
```

Comment:
+💯 
But it's true... though I don't trust much the callback data, that's why it's still there the if with a custom log statement.